app/models/group.js

```diff
@@ -15,6 +15,11 @@ exports.addModel = function(database) {
     this.screenName = params.screenName
     this.createdAt = params.createdAt
     this.updatedAt = params.updatedAt
+    if (params.hasOwnProperty('visibility')) {
```

Comment:
In FrF there are 4 group types:
- Private group (invite-only)
- Private group (subscribers can invite other people)
- Standard feed (anyone can read, only admin can post)
- Public feed (anyone can read and post).

Knowing this I'd rename "visibility" to "feedType" and convert it to integer: "0", "1", "2", and "3". Thoughts?